src/tutorials/emx/DatatypesSlide.vue

```diff
@@ -0,0 +1,144 @@
+        :data="[
+               ['string', 'Default data type, sequence of a maximum of 255 characters'],
+               ['text', 'Text of possibly more than 255 characters'],
+               ['int', 'Whole numbers between -2,147,483,648 and +2,147,483,647'],
+               ['long', 'Possibly longer whole numbers'],
+               ['decimal', 'Decimal number'],
+               ['bool', 'Boolean: TRUE/FALSE'],
+               ['date', 'Date without time according to ISO-8601'],
+               ['datetime', 'Date and time: timezone is optional but will not be stored'],
+               ['xref', 'Reference to another table presented as dropdown'],
+               ['mref', 'Reference to multiple items in another table presented as dropdown'],
+               ['categorical', 'Reference to another table presented as radio buttons'],
+               ['categoricial_mref', 'Reference to multiple items in another table presented as checkboxes'],
+               ['compound', 'Group of attributes'],
+               ['file', 'A file, needs a reference to the FileMeta table'],
+               ['email', 'An email address'],
+               ['enum', 'Item from a specified list (defined in the enumOptions in the attributes sheet)'],
+               ['script', 'Text field with code highlighting'],
+               ['html', 'Text field with HTML highlighting'],
+               ['hyperlink', 'A link to a website'],
+               ['one_to_many', 'Relation between two columns in possibly different datasets'],
+               ['text', 'Text of possibly more than 255 characters']
+               ]"/>
+        </div>
+      </div>
+    </div>
+    <div v-if="step === 2">
+      <div class="row">
+        <div class="col-12">
+          <p>How do we use these data types in EMX? Let's find out! We'll begin with
+            a small dataset:</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-8 col-sm-7">
+          <molgenis-table title="Subjects" :data="[
+            [1, 'F', 50, 60, 170, 'Indomethacin'], [2, 'M', 53, 80, 180, 'Indomethacin'],
+            [3, 'F', 53, 56, 160, 'Indomethacin'], [4, 'F', 45, 70, 175, 'Indomethacin'],
+            [5, 'M', 48, 79, 190, 'Indomethacin'], [6, 'M', 49, 70, 185, 'Indomethacin']
+          ]" :header="['id', 'sex', 'age', 'weight', 'height', 'drug']"/>
+        </div>
+        <div class="col-md-4 col-sm-5">
+          <molgenis-table title="Samples" :data="[
+              [1, 0.25, 1.5], [1, 0.5, 0.94], [1, 0.75, 0.78], [1, 1, 0.48], [1, 1.25, 0.37], [1, 2, 0.19], [1, 3, 0.12],
+              [1, 4, 0.11], [1, 5, 0.08], [1, 6, 0.07], [1, 8, 0.05], [2, 0.25, 2.03], [2, 0.5, 1.63], [2, 0.75, 0.71],
+              ['...', '...', '...']
+            ]" :header="['subject', 'time', 'conc']"/>
+        </div>
+      </div>
+    </div>
+    <div class="row" v-if="step > 2 && step < 6">
+      <div class="col-md-9 col-lg-8">
+        <p v-if="step > 2 && step < 6">
```

Comment:
I think this v-if is superfluous the state is already set on line #60

Reply:
True, I'll fix it :)